frontend/package.json

```diff
@@ -20,6 +20,7 @@
     "@types/react-router-dom": "^5.3.3",
     "buffer": "^6.0.3",
     "crypto-browserify": "^3.12.0",
+    "elliptic": "6.5.5",
```

Comment:
<img width="912" alt="image" src="https://github.com/user-attachments/assets/459efd96-a613-4c48-b275-76c36477857f">
I think the version should be larger than 6.5.6 to get it fix

Reply:
no, 6.5.6 is the latest version and it has the issue.

Reply:
so 6.5.5 is good?

Reply:
yes